### DISCUSSION TITLE
Welcome to privateGPT Discussions!

### DISCUSSION THREAD

**Discussion by imartinez** [imartinez/privateGPT#216] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

The end goal is to declutter the Issues and Pull requests sections by driving community ideas and questions to where they belong in Discussions.

**Comment by Armagedon13** [comment]
![FqRGDE6WIAM59jb](https://github.com/imartinez/privateGPT/assets/12619078/53e7c005-5e6b-44e8-9088-938c0edb99e7)

**Comment by frroossst** [comment]
~We are experimenting with this internally at work, for some of our proprietary codebases and documentation, and we're really excited about the privacy focused nature of it. What can we (as non A.I. devs) do, to make the queries faster, they take about 1-2 minute(s) for us, not quite what we're looking for though, thanks!~

conversation moved to: https://github.com/imartinez/privateGPT/discussions/223

**Reply by imartinez** [reply]
Please open a new discussion in the Discussions section and let's get the conversation going there. Thanks!

**Comment by jmbernabotto** [comment]


**Reply by imartinez** [reply]
Please open a new discussion in the Discussions section and let's get the conversation going there. Thanks!

**Comment by SubarnaKhadka** [comment]
👋

**Comment by csunny** [comment]
We posted a project which called DB-GPT,  which uses localized GPT large models to interact with your data and environment. With this solution, you can be assured that there is no risk of data leakage, and your data is 100% private and secure.

Our project is here:    https://github.com/csunny/DB-GPT
If you are interested in our project, please feel free to contact us for further communication.

**Reply by Aries-ckt** [reply]
wow what a amazing project!

**Comment by lucavehbiu** [comment]
Is there a specific python version i should use? Or is there a issue with running this on a M1 chip? I got this error:

[1]    59226 illegal hardware instruction  python privateGPT.py

**Reply by zhl3597** [reply]
I use 3.11.3, everything runs well.

**Comment by pseudotensor** [comment]
Hi guys, take a look at H2O.ai's similar project, fully Apache V2. It doesn't yet support CPU-based quantized models, but it's WIP.

https://github.com/h2oai/h2ogpt

https://github.com/h2oai/h2ogpt/pull/107 (quantization PR)

Here's a case where I pasted in text of my own
```
Yugu is a place of wonder you can visit while watching the stars.
```
which is supported via the UI

![image](https://github.com/imartinez/privateGPT/assets/2249614/55804ab8-e870-47e7-98e2-b17d21d00355)

**Reply by Martinange** [reply]
Ok

**Comment by sghosh37** [comment]
Hi @imartinez 
posted a query on the thread
https://github.com/imartinez/privateGPT/discussions/971
Can you please share some input if you are aware of the issue?
